.github/workflows/check-broken-pr-links.yml

```
@@ -0,0 +1,120 @@
+name: Check Links in Pull Requests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+
+    steps:
+      # 1️⃣ Checkout repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      # 2️⃣ Get changed Markdown files in the PR
+      - name: Get changed Markdown files
+        id: changed-files
+        run: |
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | grep '\.md$' || true)
+          CHANGED_FILES="${CHANGED_FILES//$'\n'/ }"
+          echo "CHANGED_FILES=$CHANGED_FILES" >> $GITHUB_ENV
+          echo "Changed Markdown files: $CHANGED_FILES"
+
+      # 3️⃣ Skip if no Markdown files changed
+      - name: Skip if no Markdown files changed
+        if: env.CHANGED_FILES == ''
+        run: |
+          echo "No Markdown files changed. Skipping link check."
+          exit 0
+
+      # 4️⃣ Run Lychee on changed files
+      - name: Run Lychee
+        id: run-lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          args: |
+            --no-progress
+            --include-fragments
+            --format detailed
+            ${{ env.CHANGED_FILES }}
+          output: lychee/out_raw.md
+          fail: false  # ✅ don't fail yet, let us capture output
+
+      # 5️⃣ Format Lychee output (user-friendly, relative paths)
+      - name: Format Lychee report
+        id: format-report
+        if: always()
+        run: |
+          mkdir -p lychee
+          : > lychee/comment.md   # start with empty file
+      
+          awk '
+            /^Errors in / {
+              file=$3
+              gsub("^/home/runner/work/UmbracoDocs/UmbracoDocs/", "", file)
+              print "\nFile: " file >> "lychee/comment.md"
```

Broke this into two lines for readability.
Let me know what you think.

```suggestion
              print "\nBroken links found in:\n" file >> "lychee/comment.md"
```